jsonnet/utils/set-terminationMessagePolicy.libsonnet

```diff
@@ -0,0 +1,42 @@
+{
+  setTerminationMessagePolicy(o): o {
+    local addTerminationMessagePolicy(o) = o {
+      [if std.setMember(o.kind, ['DaemonSet', 'Deployment', 'ReplicaSet']) then 'spec']+: {
+        template+: {
+          spec+: {
+            containers: [
+              c {
+                terminationMessagePolicy: 'FallbackToLogsOnError',
+              }
+              for c in super.containers
+            ],
+          } + {
+            [if 'initContainers' in super then 'initContainers']: [
```

Comment:
maybe `super` is considered to be `template` in here? (had similar behaviors before)